### ISSUE TITLE
possibility to add waypoint for projection/waypoint

### ISSUE THREAD

**Issue by DaTechniker** [cgeo/cgeo#15924]
### Tell us your idea!

There are some caches, where you need to do a projection or calculate offset from some waypoint you calculate somewhere in a multi.
Now I need to copy the coordinates for the offset and add the offset/projection

### What solution would you suggest?

Make it possible to select the waypoint the offset/projection should be done with
This would be a great benefit in usability

**Comment by Lineflyer** [comment]
Not at the app right now, but shouldn't it work to copy the waypoint you want to base the projection on and then simply go into this copied waypoint and enter distance/degree for your projection?

**Comment by DaTechniker** [comment]
Yes, this would also be possible. Sometimes I have multiple options i.e. waypoint options if i have problems with variables or took the wrong endpoint by chance. I would always need to manuell add offset/projection data again.
For me it is more a comfort feature, as I know of i.e. GCBuddy. 
I like this, but it is limited to 26 variables and migration to c:geo an PN is tricky.
Otherwise I really love the extended calculation feature of c:geo and the not existant limitation of variables and the possibility to give variables speaking names. And I like to move from using multiple apps for the same functions
